metadata.rb

```diff
@@ -4,6 +4,7 @@
 license          'All rights reserved'
 description      'Installs/Configures chef-metaflows'
 long_description 'Installs/Configures chef-metaflows'
-version          '0.1.0'
+version          '0.2.8'
 
```

Comment:
If it only works on CentOS, you could add a `supports     'centos'`. Those lines aren't enforced in a Chef run (yet), but do feed things like Supermarket that advertise what platforms are supported by what cookbooks.